.github/renovate.json5

```diff
@@ -0,0 +1,138 @@
+    "geti.com/credit_system",
+  ],
+
+  // Set limit to 5
+  ignorePresets: [":prHourlyLimit2"],
+  prHourlyLimit: 5,
+
+  packageRules: [
+    // Enable pinning for container images
+    // https://docs.renovatebot.com/presets-docker/#dockerpindigests
+    {
+      enabled: true,
+      matchDatasources: ["docker"],
+      pinDigests: true,
+      groupName: "Pin dependencies",
+      groupSlug: "pin-deps",
+    },
+
+    // Disable non-security upgrades for go, python, npm and github-actions.
+    // We will enable it in the next phase
+    {
+      enabled: false,
+      matchManagers: ["github-actions", "gomod", "pep621", "npm"],
+    },
+
+    // Disable non-security upgrades for uv, playwright and Dockerfile syntax (docker/dockerfile)
+    {
+      enabled: false,
+      matchDatasources: ["docker"],
+      matchDepNames: ["mcr.microsoft.com/playwright", "docker/dockerfile"],
+    },
+
+    // Disable python base image upgrades, except digital pinning,
+    // we will enable it in the next phase
+    {
+      enabled: false,
+      matchDatasources: ["docker"],
+      matchPackageNames: ["python"],
+      matchUpdateTypes: ["major", "minor", "patch"],
+    },
+
+    {
+      enabled: false,
+      matchDatasources: ["docker"],
+      matchPackageNames: ["docker.io/python"],
+      matchUpdateTypes: ["major", "minor", "patch"],
+    },
+
+    // Disable golang base image upgrades, except digital pinning,
+    // we will enable it in the next phase
+    {
+      enabled: false,
+      matchDatasources: ["docker"],
+      matchPackageNames: ["golang"],
+      matchUpdateTypes: ["major", "minor", "patch"],
+    },
+
+    // Disable nginx base image upgrades, except digital pinning,
+    // we will enable it in the next phase
+    {
+      enabled: false,
+      matchDatasources: ["docker"],
+      matchPackageNames: ["nginx"],
+      matchUpdateTypes: ["major", "minor", "patch"],
+    },
+
+    // Disable node image upgrades, except digital pinning,
+    // we will enable it in the next phase
+    {
+      enabled: false,
+      matchDatasources: ["docker"],
+      matchPackageNames: ["node"],
+      matchUpdateTypes: ["major", "minor", "patch"],
+    },
+
+    // Disable ghcr.io/astral-sh/uv image upgrades, except digital pinning,
+    // we will enable it in the next phase
+    {
+      enabled: false,
+      matchDatasources: ["docker"],
+      matchPackageNames: ["ghcr.io/astral-sh/uv"],
+      matchUpdateTypes: ["major", "minor", "patch"],
+    },
+
+    // Disable upgrades for non production images
+    {
+      enabled: false,
+      matchDatasources: ["docker"],
+      matchFileNames: ["**/Dockerfile.protoc"],
+    },
+  ],
+
+  // Enable security upgrades, group it in one PR
+  vulnerabilityAlerts: {
+    groupName: "Security upgrades",
+    groupSlug: "security-fix-deps",
+    enabled: true,
+    separateMajorMinor: false,
+    separateMinorPatch: false,
+  },
+  osvVulnerabilityAlerts: true,
+  dependencyDashboard: false,
```

Comment:
This dashboard seems useful to have https://docs.renovatebot.com/key-concepts/dashboard/